controllers/villains.js

```diff
@@ -0,0 +1,30 @@
+const models = require('../models')
+
+const getAllVillains = async (request, response) => {
+  const villains = await models.villains.findAll()
+
+  return response.send(villains)
+}
+
+const getVillainBySlug = async (request, response) => {
+  const { slug } = request.params
+  const matchedVillain = await models.villains.findOne({ where: { slug } })
+
+  return matchedVillain
+    ? response.send(matchedVillain)
+    : response.sendStatus(404)
+}
```

Comment:
Use  attributes to only display certain data 
const matchedVillain = await models.villains.findOne({
    where: { slug },
    attributes: ['name', 'movie', 'slug'],
  })